src/components/learner-credit-management/search/CatalogSearch.jsx

```diff
@@ -19,6 +18,7 @@ const CatalogSearch = () => {
   } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
   const searchFilters = `enterprise_catalog_uuids:${ENABLE_TESTING(subsidyAccessPolicy.catalogUuid)} AND content_type:course`;
   const displayName = subsidyAccessPolicy.displayName ? `${subsidyAccessPolicy.displayName} catalog` : 'Overview';
```

Comment:
Maybe a question for @adamstankiewicz , but is this duplicate logic from the `useBudgetDetailPage` function? 
Is there a way we can reduce duplicate logic in this case?

Reply:
@brobro10000 There most definitely is some duplicate logic here, and it's duplicated in more places than just these 2 (i.e., given a display name string, fall back to "Overview" if not set). I would recommend splitting out the refactor to DRY up the fallback to `Overview` as a separate chunk of work from this ticket, though.

Reply:
I can plus 1 on that 🦖